Codecademy/gamut
# fix(Radio): add infotip type to radio fields

### Overview


Fixes the radio-type for the new infotip prop


### PR Checklist

- [x] Related to JIRA ticket: [gm-763]

#### Testing Instructions



* Go to GridForm Page
* Gridform should render 
* Go to [this PR](https://github.com/codecademy-engineering/mono/pull/6780)
* See all the tests pass 🔥
